internal/utils/path_manager.go

```diff
@@ -18,14 +22,12 @@ func (p *PathManager) CNIServerPath() string {
 }
 
 func (p *PathManager) KubeletEndPoint() string {
-	// SockDir is the default Kubelet device plugin socket directory
-	// SockDir = "/var/lib/kubelet/plugins_registry"
 	// The following path uses the deprecated Kubelet device plugin socket directory
-	return p.wrap("/var/lib/kubelet/device-plugins/sriovNet.sock")
+	return p.wrap("/var/lib/kubelet/device-plugins/kubelet.sock")
 }
 
 func (p *PathManager) PluginEndpoint() string {
-	return p.wrap("/var/lib/kubelet/device-plugins/sriovNet.sock")
+	return p.wrap(filepath.Join("/var/lib/kubelet/device-plugins", PluginEndpointSocket))
```

Comment:
why not simply "/var/lib/kubelet/device-plugins/sriovNet.sock", like the rest?

Reply:
Because we need the name of the socket file name elsewhere and not the path:
-		Endpoint:     dp.socketFile,
+		Endpoint:     utils.PluginEndpointSocket,

Reply:
This is just the API of the device plugin that the socket name is passed without the path.

Reply:
I propose simplifying and making the code use the same style throughout (we have a similar requirement elsewhere). Elsewhere, we use the full path and then use `filepath` functions to derive a file name from the path. Otherwise, your API exposes the same (partial) information twice. The smaller the API surface, the less leaky it tends to be.

Reply:
Done.